test/integration/networks/networks_test.go

```diff
@@ -350,10 +350,10 @@ func TestNetworks(t *testing.T) {
 					// networks.DefaultVerify(assert)
 
 					servicePerimeterLink := fmt.Sprintf("accessPolicies/%s/servicePerimeters/%s", policyID, networks.GetStringOutput("restricted_service_perimeter_name"))
-					accessLevel := fmt.Sprintf("accessPolicies/%s/accessLevels/%s", policyID, networks.GetStringOutput("restricted_access_level_name"))
+					accessLevel := fmt.Sprintf("accessPolicies/%s/accessLevels/%s", policyID, networks.GetStringOutput("access_level_name"))
 					networkNames := getNetworkResourceNames(envCode, networkMode, firewallMode)
 
-					servicePerimeter := gcloud.Runf(t, "access-context-manager perimeters describe %s --policy %s", servicePerimeterLink, policyID)
+					servicePerimeter := gcloud.Runf(t, "access-context-manager perimeters dry-run describe %s --policy %s", servicePerimeterLink, policyID)
```

Comment:
```suggestion
					servicePerimeter, err := gcloud.RunCmdE(t, fmt.Sprintf("access-context-manager perimeters dry-run describe %s --policy %s", servicePerimeterLink, policyID))
```

It will be necessary to use `gcloud.RunCmdE` instead of `gcloud.Runf` since the output of `perimeters dry-run describe` is a [diff](https://cloud.google.com/sdk/gcloud/reference/access-context-manager/perimeters/dry-run/describe#:~:text=The%20dry%2Drun%20mode%20configuration%20is%20presented%20as%20a%20diff%20against%20the%20enforcement%20mode%20configuration) and it is not in json format

To verify results  strings.Contains can be used like 
```
strings.Contains(servicePerimeter, servicePerimeterLink) 
```
the result of the gcloud command is 

```
  name: sp_n_shared_restricted_default_perimeter_f230
  title: sp_n_shared_restricted_default_perimeter_f230
  type: PERIMETER_TYPE_REGULAR
+ accessLevels:
+   accessPolicies/REDACTED1/accessLevels/alp_n_shared_restricted_members_dry_run_f230
+ resources:
+   projects/REDACTED2
+ restrictedServices:
+   gkehub.googleapis.com
+   notebooks.googleapis.com
+   networksecurity.googleapis.com
+   meshconfig.googleapis.com
+   servicedirectory.googleapis.com
+   cloudbuild.googleapis.com
+   managedidentities.googleapis.com
+   bigqueryreservation.googleapis.com
+   firebaserules.googleapis.com
+   sts.googleapis.com
+   containerfilesystem.googleapis.com
+   alpha-documentai.googleapis.com
+   privateca.googleapis.com
+   dlp.googleapis.com
+   networkconnectivity.googleapis.com
+   networkmanagement.googleapis.com
+   assuredworkloads.googleapis.com
+   integrations.googleapis.com
+   iam.googleapis.com
+   oslogin.googleapis.com
+   apigeeconnect.googleapis.com
+   vpcaccess.googleapis.com
+   secretmanager.googleapis.com
+   metastore.googleapis.com
+   containerregistry.googleapis.com
+   adsdatahub.googleapis.com
+   binaryauthorization.googleapis.com
+   clouddebugger.googleapis.com
+   ids.googleapis.com
+   bigquerymigration.googleapis.com
+   eventarc.googleapis.com
+   file.googleapis.com
+   orgpolicy.googleapis.com
+   iaptunnel.googleapis.com
+   retail.googleapis.com
+   pubsub.googleapis.com
+   cloudkms.googleapis.com
+   gkebackup.googleapis.com
+   compute.googleapis.com
+   tpu.googleapis.com
+   containeranalysis.googleapis.com
+   domains.googleapis.com
+   cloud.googleapis.com
+   networkservices.googleapis.com
+   webrisk.googleapis.com
+   firestore.googleapis.com
+   contactcenterinsights.googleapis.com
+   recaptchaenterprise.googleapis.com
+   aiplatform.googleapis.com
+   sqladmin.googleapis.com
+   healthcare.googleapis.com
+   dataproc.googleapis.com
+   analyticshub.googleapis.com
+   pubsublite.googleapis.com
+   composer.googleapis.com
+   recommender.googleapis.com
+   dns.googleapis.com
+   containerthreatdetection.googleapis.com
+   timeseriesinsights.googleapis.com
+   gkeconnect.googleapis.com
+   redis.googleapis.com
+   lifesciences.googleapis.com
+   osconfig.googleapis.com
+   alloydb.googleapis.com
+   cloudtrace.googleapis.com
+   monitoring.googleapis.com
+   vision.googleapis.com
+   opsconfigmonitoring.googleapis.com
+   bigquery.googleapis.com
+   run.googleapis.com
+   clouderrorreporting.googleapis.com
+   bigquerydatatransfer.googleapis.com
+   automl.googleapis.com
+   logging.googleapis.com
+   cloudasset.googleapis.com
+   vmmigration.googleapis.com
+   clouddeploy.googleapis.com
+   datamigration.googleapis.com
+   texttospeech.googleapis.com
+   baremetalsolution.googleapis.com
+   iamcredentials.googleapis.com
+   visionai.googleapis.com
+   ml.googleapis.com
+   cloudresourcemanager.googleapis.com
+   cloudsearch.googleapis.com
+   workflows.googleapis.com
+   container.googleapis.com
+   language.googleapis.com
+   datacatalog.googleapis.com
+   workstations.googleapis.com
+   apigee.googleapis.com
+   gameservices.googleapis.com
+   bigtable.googleapis.com
+   storage.googleapis.com
+   cloudfunctions.googleapis.com
+   datafusion.googleapis.com
+   datastream.googleapis.com
+   documentai.googleapis.com
+   meshca.googleapis.com
+   cloudscheduler.googleapis.com
+   dataplex.googleapis.com
+   servicecontrol.googleapis.com
+   connectgateway.googleapis.com
+   accessapproval.googleapis.com
+   transcoder.googleapis.com
+   batch.googleapis.com
+   krmapihosting.googleapis.com
+   firebaseappcheck.googleapis.com
+   spanner.googleapis.com
+   videointelligence.googleapis.com
+   cloudprofiler.googleapis.com
+   memcache.googleapis.com
+   dataflow.googleapis.com
+   storagetransfer.googleapis.com
+   bigquerydatapolicy.googleapis.com
+   kmsinventory.googleapis.com
+   dialogflow.googleapis.com
+   artifactregistry.googleapis.com
+   trafficdirector.googleapis.com
+   speakerid.googleapis.com
+   translate.googleapis.com
+   speech.googleapis.com
  vpcAccessibleServices:
+   allowedServices:
+     RESTRICTED-SERVICES
    enableRestriction: true
[]
```

Reply:
Thanks Daniel, initially I suspected the different format between enforced and dry-run was a bug in gcloud and raised a bug to VPCSC product team. (It's unusual that the gcloud commands for dry-run ignore all the --format flags that normally have global supprt). But I overlooked that it's a diff, so the different format is probably WAI.

I'll try modifying the tests to accept the new format as you suggested